src/common/utils/date/get-current-date-time.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import moment from "moment";
+
+export const getCurrentDateTime = () => moment().utc().format();
```

Comment:
I'd like to say yes, but previous experience says that it's best to override `Date` by messing with globals, because of the all of the libraries that use it. If we used `injectable` for it, then we would have absolute control in our code base, but if (and when) we use some library that uses `Date.now`, we will get separate opinion which is easy way encounter hard-to-debug failing unit tests.

Reply:
Fair enough, thought after talking with @nevalla it seems that the timestamp is received automatically, so we don't need to include it.

Reply:
I'd say that let's keep it there. Sometimes it's easier to access data from event itself, and not from the metadata. It all depends from the choice of service that we are using.